hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/store/contracts/precompile/HTSPrecompiledContract.java

```diff
@@ -313,7 +314,7 @@ void prepareFields(final MessageFrame frame) {
     void prepareComputation(Bytes input, final UnaryOperator<byte[]> aliasResolver) {
         this.precompile = null;
         this.transactionBody = null;
-
+        validateTrue(input.size() >= 4, INVALID_TRANSACTION_BODY);
```

Comment:
Adding on to @tinker-michaelj comments above, can we not simple return `null` to accomplish the same results?

Reply:
You are right. Are we talking only about the `HtsPrecompliledContract` here?

Reply:
Yes, just in the `HtsPrecompliledContract`.  I did have a question that I posted below which is were these new checks necessary to show that transfers to system contracts would fail?